surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformIT.java

```diff
@@ -40,27 +41,24 @@ public void setUp()
     public void testJupiterEngine()
     {
         unpack( "/junit-platform-engine-jupiter" )
+                .setTestToRun( "Basic*Test" )
                 .executeTest()
                 .verifyErrorFree( 5 );
     }
 
     @Test
-    @Ignore( "Uncomment while developing SUREFIRE-1222. Rename 'javax' extension of DisplayNameTest.javax." )
```

Comment:
This change seems unrelated

Reply:
This was the main test for this change.
The DisplayName is permanently enabled in this commit which is still not good because the user should switch it on/off in the configuration. So therefore we are waiting for the extensions and config changes. Basically this test tells us that the strings with display name and good encoding is in TXT and XML reports however still permanently enabled.

Reply:
Got it.
Sounds good to me